styles/modules/modals/_editListing.scss

```diff
@@ -35,26 +56,59 @@
               margin-bottom: 0;
             }
 
-            .closeIcon {
-              width: 24px;
-              height: 24px;
-            }
-
             &.addElement {
               .imagesIcon {
                 font-size: 50px;
                 @include center;
                 top: 35%;
               }
+            }
+          }
+        }
+      }
+
+      .shippingSection {
+        .tile {
+          .locationIcon {
+            font-size: 38px;
+            @include center;
+            top: 34%;
+          }
+        }
+
+        &.hasShippingOptions {
+          .emptyShippingOptionsSection {
+            display: none;
+          }
+        }
+
+        form {
+          label {
+            font-weight: 400;          
+          }
+
+          .shippingOptionButtonBar {
+            margin-top: 25px;
+          }
+
+          .servicesWrap {
```

Comment:
You could take out some of this by using flexColRows and gutterV instead, since that's designed to give a vertical stack with no padding on the sides.